plugins/modules/rundeck_acl_policy.py

```diff
@@ -36,6 +36,12 @@
         description:
             - Sets the project name.
         required: true
+    api_token:
+        type: str
+        description:
+            - Sets the token to authenticate against Rundeck API.
+        required: true
```

Comment:
Instead of overwriting everything, I would only overwrite the different fields:
```suggestion
```

Reply:
Oh, I see!
Didn't know that it is possible do it, thanks.